### PR TITLE
updated JES uncertainties to RunIIFall17 version,

### PR DESCRIPTION
**Pull request by veelken**
downloaded from https://twiki.cern.ch/twiki/bin/viewauth/CMS/JECDataMC

### CODE REVIEW

File: python/postprocessing/modules/jme/jetmetUncertainties.py [hunk 1]
```diff
@@ -32,8 +32,9 @@ def __init__(self, globalTag, jesUncertainties = [ "Total" ], jetType = "AK4PFch
         # read jet energy scale (JES) uncertainties
         self.jesInputFilePath = os.environ['CMSSW_BASE'] + "/src/PhysicsTools/NanoAODTools/data/jme/"
         if len(jesUncertainties) == 1 and jesUncertainties[0] == "Total":
-            self.jesUncertaintyInputFileName = "Summer16_23Sep2016V4_MC_Uncertainty_AK4PFchs.txt"
+            self.jesUncertaintyInputFileName = "Fall17_17Nov2017_V4_MC_Uncertainty_AK4PFchs.txt"
```
Comment (arizzi) [inline python/postprocessing/modules/jme/jetmetUncertainties.py]: we cannot do it this way, we must support 2016 and 2017 (same as for b-tag)

btw please note that there is a PR from Sal restructuring a bit this stuff... so I'm not sure if this is going to conflict

**Comment by veelken** [comment]
Hi Andrea,

I have followed the procedure we converged-on for the btagSF tool and added an era parameter to switch between 2016 and 2017 JEC corrections and uncertainties.
Please have  a look at the new version and let me know if this PR can be merged now.

Cheers,

Christian

**Comment by arizzi** [comment]
as kind of expected we have conflicts after I merged Sal's PR

**Comment by veelken** [comment]
I have updated this PR according to the changes implemented by Sal in PR #48. Please merge this PR now.

**Comment by veelken** [comment]
Hi, 

would be great if this PR can be merged now.

Cheers,


Christian

**Pull request merged**